app/apollo/resolvers/channel.js

```diff
@@ -460,6 +463,9 @@ const channelResolvers = {
         // Save the change
         await models.Channel.updateOne({ org_id, uuid }, { $set: { name, tags, custom, remote, updated: Date.now() } }, {});
 
+        // Save previous name for qraphQL plugins
+        const previousName = channel.name;
```

Comment:
Why save here and use later rather than just (later) doing
```
        // Allow graphQL plugins to retrieve more information
        context.pluginContext = {name, previousName: channel.name, uuid: uuid};
```
?  It doesnt look like `channel.name` would somehow _change_ between the two spots, so explicitly saving the value into a new variable here just adds complexity.

Reply:
You're right, I think I was thinking about each of the other functions and how they all keep context.pluginContext in the same area in code above the return.